include/aspect/utilities.h

```diff
@@ -156,6 +156,11 @@ namespace aspect
     bool fexists(const std::string &filename);
 
     /**
+     * Retrieve the last line of a file
+     */
+    std::string get_last_line(std::ifstream *in);
```

Comment:
Sure, I understand your point that there is no advantage of using `ifstream` as opposed to the `istream.`